cpp/cmake/config.json

```diff
@@ -0,0 +1,50 @@
+{
+    "parse": {
+        "additional_commands": {
+            "CPMFindPackage": {
+                "kwargs": {
+                    "NAME": 1,
+                    "GITHUB_REPOSITORY": "?",
+                    "GIT_TAG": "?",
+                    "VERSION": "?",
+                    "GIT_SHALLOW": "?",
+                    "OPTIONS": "*",
+                    "FIND_PACKAGE_ARGUMENTS": "*"
+                }
+            },
+            "rapids_cython_create_modules": {
+                "kwargs": {
+                    "SOURCE_FILES": "?",
+                    "LINKED_LIBRARIES": "?",
+                    "MODULE_PREFIX": "?",
+                    "ASSOCIATED_TARGETS": "?"
+                }
+            }
+        }
```

Comment:
Telling `cmake-format` the signature of this function helps it to split keyword arguments out one per line.